tests/inlineeditorui.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals document, Event */
+
+import ComponentFactory from '@ckeditor/ckeditor5-ui/src/componentfactory';
+import View from '@ckeditor/ckeditor5-ui/src/view';
+
+import InlineEditorUI from '../src/inlineeditorui';
+import InlineEditorUIView from '../src/inlineeditoruiview';
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+
+import FocusTracker from '@ckeditor/ckeditor5-utils/src/focustracker';
+import * as enableToolbarKeyboardFocus from '@ckeditor/ckeditor5-ui/src/toolbar/enabletoolbarkeyboardfocus';
```

Comment:
OK, I mistakenly merged the PR in editor-classic which has exactly the same issue: https://github.com/ckeditor/ckeditor5-editor-classic/pull/50/files/7377a8021ddf03236599622e9e768058e21df18f#r104738910. Please, fix both at the same time.